README.md

```diff
@@ -53,6 +179,8 @@ make run_mcmc
 cd ..
 ```
 
+--->
+
 ### Tutorial
```

Comment:
I would upgrade this header to only one #, this has no business in being in the building section